uptane-standard.md

```diff
@@ -281,10 +291,9 @@ The following topics will not be addressed in this document, as they represent t
 
 ## Design Requirements
 
-The design requirements for this document are governed by three principal parameters:
+The design requirements for this document are governed by the following principal parameters:
 
 * to clearly mandate the design and implementation steps that are security critical and must be followed as is, while offering flexibility in the implementation of non-critical steps. In this manner, users can adapt to support different use models and deployment scenarios.
-* to delineate best practices to ensure that, should a vehicle be attacked, an attacker must compromise many different systems to control the vehicle, access sensitive data, or seriously impact functionality.
```

Comment:
What was the problem with this sentence? The compartmentalization of Uptane is one of its features, or at least, it was a fundamental aspect of the design.